test/parallel/test-vm-cached-data.js

```diff
@@ -28,6 +29,15 @@ function produce(source, count) {
       if (script.cachedDataProduced)
         data = script.cachedData.toString('base64');
     }
+
+    for (var i = 0; i < ${count}; i++) {
+      var script = new vm.Script(process.argv[1]);
+      const cachedData = script.createCachedData();
+
+      assert(cachedData instanceof Buffer);
+
+      data = cachedData.toString('base64');
```

Comment:
This overwrites `data` created earlier. Is it possible to test the buffer returned from both `produceCachedData` and `createCachedData`?